sklearn/linear_model/_ridge.py

```diff
@@ -2189,6 +2226,16 @@ def fit(self, X, y, sample_weight=None):
         alphas = np.asarray(self.alphas)
 
         if cv is None:
+            if _routing_enabled():
+                routed_params = process_routing(
+                    self,
+                    "fit",
+                    sample_weight=sample_weight,
+                    **params,
+                )
+            else:
+                routed_params = Bunch(scorer=Bunch(score={}))
```

Comment:
I would expect to pack `sample_weight` in the `routed_params`. Isn't it a bug similar to `LogsisticRegressionCV` to not include the weights in the `scorer`?

Reply:
If routing is not enabled do we pass sample_weight in the scorer? I think since currently it is not being passed maybe we should leave it?

Reply:
But this seems to be rather a bug similarly to the one in `LogisticRegressionCV` where we fixed it.

Reply:
I see. So should I add it then?

Reply:
I'm agnostic on whether to pass sample_weight to the scorer here or not, happy either way. But if we're doing it, we should make sure to add a FIX entry to the changelog.